iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/rest/IcebergTableOperations.java

```diff
@@ -82,7 +82,7 @@ public class IcebergTableOperations {
   private ObjectMapper icebergObjectMapper;
   private IcebergTableOperationDispatcher tableOperationDispatcher;
 
-  @Context private HttpServletRequest httpRequest;
+  @Inject private Provider<HttpServletRequest> httpRequest;
```

Comment:
Curious why is this needed?

Reply:
`TestTableOperations` failed because httpRequest is null ,   I try to fix the test failure by using `Inject`, but still not fixed.